application/app/Http/Resources/ClassifierValueResource.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Http\Resources;
+
+use App\DataTransferObjects\Meta\LanguageMetaData;
+use App\DataTransferObjects\Meta\ProjectTypeMetaData;
+use App\Enums\ClassifierValueType;
+use App\Models\ClassifierValue;
+use Illuminate\Http\Request;
+use Illuminate\Http\Resources\Json\JsonResource;
+
+/**
+ * @mixin ClassifierValue
+ */
+class ClassifierValueResource extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @return array<string, mixed>
+     */
+    public function toArray(Request $request): array
+    {
+        return [
+            'id' => $this->id,
+            'name' => $this->name,
+            'value' => $this->value,
+            'type' => $this->type->value,
+            'meta' => $this->getMetaData($this),
```

Comment:
**[non-blocker]** I’m not sure whether the `getMetaData()` method is necessary. I think the same functionality could be achieved with the following:

```suggestion
            'meta' => $this->meta
```

Is there some other reason why the `getMetaData()` method is necessary? I’d say: "less code, the better".